pymemcache/test/test_serde.py

```diff
@@ -59,3 +61,18 @@ def test_pickleable(self):
     def test_subtype(self):
         # Subclass of a native type will be restored as the same type
         self.check(CustomInt(123123), FLAG_PICKLE)
+
+
```

Comment:
Worth adding a distinct test case for `pickle.HIGHEST_PROTOCOL`, too? It's redundant with `2`, but not in the "test the full interface" sense.